### ISSUE TITLE
Write integration test which exercises libfontmanager.a

### ISSUE THREAD

**Issue by jerboaa** [Karm/mandrel-integration-tests#16]
Similar to the ImageIO integration test work, but using some code which actually exercises libharfbuzz.so code too (which previously was included in libfontmanager.so and is a separate library on some systems - or a system library). One test that fits the bill is [RotatedFontTest.java from OpenJDK 11](https://hg.openjdk.java.net/jdk-updates/jdk11u-dev/file/4ed322bf6b00/test/jdk/java/awt/font/Rotate/RotatedFontTest.java). This would ensure that generating native images from fonts-using code won't regress.

**Comment by jerboaa** [comment]
@Karm One way to verify that libfontmanager.a and libharfbuzz.a are being pulled in is to use `-Dquarkus.native.additional-build-args="-H:+TraceNativeToolUsage"` option and look for those libs on the link command.

**Comment by jerboaa** [comment]
FWIW, I've verified using gdb that it stops in harfbuzz code like so:
```
$ gdb -ex 'set breakpoint pending on' -ex 'break hb_face_make_immutable'  -ex 'run' --args ./font-rotate-test
[...]
Reading symbols from ./font-rotate-test...
Breakpoint 1 at 0xeb5de0: file /builddir/build/BUILD/java-11-openjdk-11.0.10.0.9-1.fc33.x86_64/openjdk/src/java.desktop/share/native/libharfbuzz/hb-face.cc, line 331.
Starting program: /home/sgehwolf/Documents/mandrel/bugs/MANDREL-37/harfbuzz_repro/font-rotate-test 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
[New Thread 0x7ffff632b640 (LWP 320150)]
java.awt.Font[family=Monospaced,name=Monospaced,style=plain,size=20]
Font:java.awt.Font[family=Monospaced,name=Monospaced,style=plain,size=20] GTX:AffineTransform[[1.0, 0.0, 0.0], [0.0, 1.0, 0.0]] FM:Integer text metrics mode using drawString

Thread 1 "font-rotate-tes" hit Breakpoint 1, hb_face_make_immutable (face=0x19fcd80)
    at /builddir/build/BUILD/java-11-openjdk-11.0.10.0.9-1.fc33.x86_64/openjdk/src/java.desktop/share/native/libharfbuzz/hb-face.cc:331
331 /builddir/build/BUILD/java-11-openjdk-11.0.10.0.9-1.fc33.x86_64/openjdk/src/java.desktop/share/native/libharfbuzz/hb-face.cc: No such file or directory.
```

Where `./font-rotate-test` is a native image of `RotatedFontTest.java`

**Comment by Karm** [comment]
@jerboaa  Thx. I was about to drop you a message.  I tried browsing symbols in the binary looking for *hb_* to see harfbuzz functions, e.g. ` objdump -x target/imageio | grep hb
` if they were baked into it, also things like `strace target/imageio 2>&1 | grep harf` to see if it wasn't loaded at runtime...
And I failed to see any `hb_` with the app (which is almost what RotatedFontTest) does.   Gonna try with gdb as you suggested...
